package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "qwixx_game",
```

Comment:
Did you  add another package-lock.json in the parent directory of "client" & "server"?

Reply:
oh this may have been error when doing some git stuff. Im not sure whats happened there